src/queries/auth/authMutations.ts

```diff
@@ -1,9 +1,40 @@
 import { UseMutationOptions, useMutation } from 'react-query';
-import { signInUser } from '@/data/auth';
-import { SignInUserData, SignInUserResponse } from '@/types/auth';
+import {
+  signInUser,
+  signUpUser,
+  refreshToken,
+  logoutUser,
+} from '../../data/auth';
+import {
+  SignInUserData,
+  SignInUserResponse,
+  SignUnUserResponse,
+  SignUpUserData,
+  RefreshTokenData,
+  RefreshTokenResponse,
+  LogoutUserResponse,
+} from '../../types/auth';
 
 export function useSignInUserMutation(
   options?: UseMutationOptions<SignInUserResponse, Error, SignInUserData>,
 ) {
   return useMutation('sign-in-user-mutation', signInUser, options);
 }
+
+export function useSignUnUserMutation(
+  options?: UseMutationOptions<SignUnUserResponse, Error, SignUpUserData>,
+) {
+  return useMutation('sign-in-user-mutation', signUpUser, options);
+}
+
+export function useRefreshTokenMutation(
```

Comment:
You can delete this mutation.Since service `refreshToken` won't be called in some part of app except a response interceptor